config/galaxy.yml.sample

```diff
@@ -998,6 +998,12 @@ galaxy:
   # <project_name> -> Settings -> API Keys.
   #sentry_dsn: null
 
+  # Sentry slow request logging.  Requests slower than the threshold indicated
+  # below will be sent as events to the configured Sentry server (above).  A
+  # value of '0' is disabled.  For example, you would set this to .005 to log
+  # all queries taking longer than 5 milliseconds.
+  #sentry_sloreq_threshold: 0
+
```

Comment:
Reminder that this needs to be moved to `lib/galaxy/webapps/galaxy/config_schema.yml` . xref https://github.com/galaxyproject/galaxy/issues/5483